0_example/todo/main.go

```diff
@@ -44,6 +44,7 @@ func main() {
 		m.Command("add", t.addHandler)
 		m.Command("rm", t.removeHandler)
 		m.Command("list", t.listHandler)
+		m.Autocomplete("rm", t.autoCompleteNames)
```

Comment:
I wonder if we should just build autocomplete into `Command`, that way we don't need to double up on the router.

Reply:
as in, make command
```go
func (m *Mux) Command(route string, handler Handler)
```
into
```go
func (m *Mux) Command(route string, handler Handler, autocomplete ...Handler)
``` and use those autocomplete handlers?

I'm not sure how I feel about that, especially since we can only mount one handler per route per tree.

It would make the API kind of weird, I think

Reply:
Actually, this could use `m.Route("rm")` to group them, so maybe it's not a big deal